client-sdk/ts-web/core/src/client.ts

```diff
@@ -569,7 +574,15 @@ export class GRPCWrapper {
     ): Promise<RESP> {
         // @ts-expect-error missing declaration
         const name = desc.name;
-        return this.client.thenableCall(this.base + name, request, null, desc).catch((e) => {
+        const method = this.base + name;
+        // Some browsers with enormous market share aren't able to preserve the stack between here
+        // and our `.catch` callback below. Save a copy explicitly.
+        // In particular, our build process rewrites async code as some generator thing, and that
+        // breaks those aforementioned browsers' tracking of async stacks. If in the future we
+        // start shipping native async/await code, let's remove this and conver callUnary to
```

Comment:
```suggestion
        // start shipping native async/await code, let's remove this and convert callUnary to
```

Reply:
huh, although tangentially even without converting these to async we already get to see all the way out to `runCallEffect`

Reply:
oh shoot, its probably not that, with the bottom of the extra stack mixing together with the real stack

Reply:
this turned out to be wrong anyway. I'll remove the comment